template.yml

```diff
@@ -292,7 +292,7 @@ Resources:
       BucketName: !If [IsDevCondition, !Sub "cdo-dev-${SubDomainName}-output", !Sub "cdo-${SubDomainName}-output"] 
       CorsConfiguration:
         CorsRules:
-          - AllowedMethods: [GET]
+          - AllowedMethods: [GET, PUT]
```

Comment:
does this allow anyone to put or is there auth on this?

Reply:
As far as I understand, this should require the same level of permissions that are required for a GET request. Testing this in postman, if I try to make a PUT request to the location without any pre-signed info (just https://javabuilder-sanchit.dev-code.org/sessionId/file) I get an access denied, but with a pre-signed URL, I can upload.

Reply:
Yep, this just tells the browser it's okay to attempt the request, but server side auth will still apply. https://docs.aws.amazon.com/AmazonS3/latest/userguide/cors.html